dev/CommonStyles/Button_themeresources.xaml

```diff
@@ -120,7 +120,9 @@
 
     <Thickness x:Key="ButtonPadding">8,5,8,6</Thickness>
 
-    <Style TargetType="Button">
+    <Style TargetType="Button" BasedOn="{StaticResource DefaultButtonStyle}" />
+
+    <Style x:Key="DefaultButtonStyle" TargetType="Button">
```

Comment:
@kaiguo would apps that consume WinUI be able to create styles based on these styles?

Reply:
That's the idea! If you want to give it a spin you can try the nupkg from the PR build. Try this link?

https://dev.azure.com/ms/_apis/resources/Containers/3023874?itemPath=drop%2FMicrosoft.UI.Xaml.2.3.190910001-prerelease.nupkg

Reply:
Yes, apps consuming WinUI should be able to use these style. The link Jevan posted above has the latest changes, you can give that a try : )

Reply:
Just tried it and indeed it works! Thanks.

Will there be a docs page or something that lists all the style key names so that we know what key name to put in the `BasedOn` style property when creating our own styles?

Reply:
> Just tried it and indeed it works! Thanks.
> 
> Will there be a docs page or something that lists all the style key names so that we know what key name to put in the `BasedOn` style property when creating our own styles?

As well as docs, will the Design Time experience allow creating copies of all templates in VS or Blend?

Reply:
@jevansaks who should we ping for doc updates?

Reply:
We can just make the changes, but @kikisaints @chigy should recommend which doc we would add it to.